notebooks/notebook_template.ipynb

```diff
@@ -31,6 +31,8 @@
       "source": [
```

Comment:
<p>This command only lists the contents, and throws an error if it doesn't exist: </p><p>E.g., it returns "BucketNotFoundException: 404 gs://polong-deleteme-your-bucket-name-unique bucket does not exist."</p><p> </p><p>This should instead be:</p><p>! gcloud storage buckets create $BUCKET_URI</p><p> </p><p>If a user runs it for the first, they see:</p><p>Creating gs://polong-deleteme-your-bucket-name-unique/...</p><p> </p><p>Then if they attempt to re-run it they will see:</p><pre class="ql-syntax" spellcheck="false">Creating gs://polong-deleteme-your-bucket-name-unique/...
ERROR: (gcloud.storage.buckets.create) HTTPError 409: Your previous request to create the named bucket succeeded and you already own it.
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>I think we can remove this line actually.</p>